export-image/04-finalise/01-run.sh

```diff
@@ -15,8 +15,10 @@ fi
 rm -f ${ROOTFS_DIR}/etc/apt/apt.conf.d/51cache
 rm -f ${ROOTFS_DIR}/usr/sbin/policy-rc.d
 rm -f ${ROOTFS_DIR}/usr/bin/qemu-arm-static
-if [ -e ${ROOTFS_DIR}/etc/ld.so.preload.disabled ]; then
-        mv ${ROOTFS_DIR}/etc/ld.so.preload.disabled ${ROOTFS_DIR}/etc/ld.so.preload
+if [ "$USE_QEMU" = "false" ]; then
```

Comment:
Shouldn't this be a double equals?

Reply:
Should it? Everything I've seen suggests that = and == mean the same thing (string comparison).

Reply:
And I undestand this like:
If we are not in qemu mode then proceed as usual

Reply:
In bash it works with just an single equal

Reply:
The bash reference manual says '=' and '==' are synonymous.

http://tldp.org/LDP/abs/html/comparison-ops.html

Reply:
In fact, trying a `==` in dash seems to error out, so it looks like `=` is more portable, but it doesn't matter here.

Reply:
indeed, this feature is disabled by default.